packages/api/proto/regen/data/v1/query.proto

```diff
@@ -16,7 +16,7 @@ service Query {
   rpc AnchorByIRI(QueryAnchorByIRIRequest) returns (QueryAnchorByIRIResponse) {
     option (google.api.http) = {
       get : "/regen/data/v1/anchor-by-iri/{iri}"
-      additional_bindings : [ {get : "/regen/data/v1/anchors/iri/{iri}"} ]
+      // additional_bindings : [ {get : "/regen/data/v1/anchors/iri/{iri}"} ]
```

Comment:
Telescope was returning unexpected `[` character error. Additional bindings are not necessary.

Reply:
Correct. Eventually we shouldn't have to make any changes to the proto files and we could use buf to pull the proto files from buf registry and gitignore proto files. This would require adding a fix on telescope though as well as fixing the proto file with the extra `;` (or adding a fix on telescope for this as well).